server/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -557,6 +573,82 @@ public String toString() {
         }
     }
 
+    /**
+     * Extension of the Clusters class for CCS uses. It adds the number of remote clusters
+     * included in a search and whether ccs_minimize_roundtrips is true.
+     *
```

Comment:
not that it matters a lot, but IDEA gives me a warning that these blank lines will be ignored. Maybe we can remove them then?